caddy/content.md

```diff
@@ -20,6 +20,14 @@ In the examples below, a [named volume](https://docs.docker.com/storage/volumes/
 
 Note that named volumes are persisted across container restarts and terminations, so if you move to a new image version, the same data and config directories can be re-used.
 
+### Linux capabilities
```

Comment:
I don't think `Linux capabilities` should be _above_ `Basic Usage`. I think it should go later in the document, because it's not absolutely necessary to run Caddy, it's just a nice-to-have for HTTP/3 performance.

Reply:
I wasn't sure where to put it, to be honest. Some security-conscious people may want to see this notice in the top part, but I agree that Basic Usage should go first here.

Since the automatic TLS example would already grant NET_ADMIN, should we put the section in front of it?

Reply:
I think the point of this document is to explain _how to use the image_ and details like this are less important and should go later in the document.

I wish Docker Hub had proper markdown anchor tags like every other modern markdown renderer because then we could just add a link to jump to that section from the TLS section which mentions it. https://github.com/docker-library/docs/issues/2304 and https://github.com/docker/hub-feedback/issues/956